### PR TITLE
Implement a blog

### LINKED ISSUE #46: Implement a blog
We need to add a blog on our website.

For now it will be in English only but copied to both the FR & EN sites (like the API documentation).

The HTML had already been done a while ago (in 2017): https://github.com/HubRise/hubrise-marketing-site

The 2 pages we need to add are: `build/article.html` and `build/blog.html`

No need for a comment section. Instead we will simply add an expendable box at the bottom "Comment this article", with the same style as the feedback component: 
image
When the block is expanded, it will propose to send us an email or share a comment on Twitter (not sure about the exact copy yet)

Also, what do you think about the search box? Is that doable in Gatsby?

### PR DESCRIPTION
**Pull request by antmonnier**
Closes #46

### CODE REVIEW

**Comment by antmonnier** [comment]
Hi @Ozerich 

Very good job!

Here are my comments:

- CSS should be kept in SCSS files for now. I understand that the trend is to move CSS in JS but I first need to educate myself on this (didn't have the time yet) and then take a decision for the whole project rather than mixing the 2 paradigms.

- We use `rem` in all our CSS. We use `px` rarerly, only when you have to (ie you need a 1px border). We never use `em`. Better revert to the original CSS, as I could see a few other issues with your CSS (for example https://i.imgur.com/fTwaTCm.png or https://i.imgur.com/PiTN9WX.png) and the original CSS has already been approved.

- Wrong color here, should either be gray (or removed if you cannot easily style it): 
image

- There won't be author pages. You can make author names as simple text instead of links.

- A few properties in the 2 example blog posts frontmatters are not used, please remove them

- These links should be dynamically generated: 
image
Use reverse chronological order on articles post dates, include every month where at least one article was posted within the current year, then the past years. So assuming we started blogging in 2018 and did not post an article on Feb 2020, it would look like this:
```
March 2020
January 2020
2019
2018
```

- These 2 links should not be foldable on desktop: 
image

- Images in /blog should be clickable and link to the post

- The last item in the breadcrumb should not be a link 
image

- The feedback section should have a different copy (but keep the same style) from the documentation pages. Title should be "Feedback on this article", text can be "Lorem ipsum" for now (will edit myself later)

That is all for now. I'll probably add more items later, after the original CSS has been restored.

Thanks for the good work so far.

**Comment by Ozerich** [comment]
All issues are fixed except archives. I'm going to implement it tomorrow

File: src/components/blog/feedback.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          onClick={() => setIsExpanded(!isExpanded)}
+        >
+          <button className="feedback__title-icon">
+            <FontAwesomeIcon
+              icon={faAngleDown}
+              flip={isExpanded && `vertical`}
+              fixedWidth
+            />
+          </button>
+          <p className="feedback__title">Feedback on this article</p>
```
Comment (antmonnier) [inline src/components/blog/feedback.jsx]: Must be localizable

File: src/components/blog/post.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            <GatsbyImage fixed={image} />
+          </div>
+        ) : (
+          <Link to={url} className="articles__image-block">
+            <GatsbyImage fixed={image} />
+          </Link>
+        )}
+        <div className="articles__description-block">
+          <p className="articles__date">
+            Posted on{' '}
```
Comment (antmonnier) [inline src/components/blog/post.jsx]: All constant strings must be localizable

File: src/components/blog/feedback.jsx (hunk 1 above)
Comment (antmonnier) [inline src/components/blog/feedback.jsx]: Must be localizable

File: src/components/blog/sidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
+import React, { useEffect, useState } from 'react'
+import { Link } from 'gatsby'
+import cx from 'classnames'
+import { useMedia } from 'react-use'
+
+function getRecentPosts(postList) {
+  return [...postList].sort((a, b) => b.date - a.date).slice(0, 5)
+}
+
+const monthList = [
```
Comment (antmonnier) [inline src/components/blog/sidebar.jsx]: Must be localizable

File: content/blog/en/2020-01-28_the-first-blog-article.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+layout: article
+meta:
+  title: Article
+  description:
+
+title: Title of first blog article
+date: 2020-01-28
+shortDescription: |
+  Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+description: |
```
Comment (antmonnier) [inline content/blog/en/2020-01-28_the-first-blog-article.md]: `description` does not seem to be used

File: src/components/blog/feedback.jsx (hunk 1 above)
Comment (antmonnier) [reply src/components/blog/feedback.jsx]: EN copy:
### Feedback on this article
Comments on this article are welcome. Don't hesitate to share your thoughts [by sending us a message]. We will reply to you.

FR copy:
### Commentaires sur cet article
Les commentaires sont les bienvenus. N'hésitez pas à nous faire part de vos remarques [en nous envoyant un message]. Nous vous répondrons.

**Comment by Ozerich** [comment]
I've fixed all bugs and implemented blog archive

**Comment by Ozerich** [comment]
Fixed

**Comment by antmonnier** [comment]
Great stuff!

Only thing I could spot is that we miss a search box here: 
image

Is there a technical reason? Can you easily add it?

**Comment by Ozerich** [comment]
I've added search box. Check please

**Comment by antmonnier** [comment]
Thanks, also just realized that mobile display of the article page was broken - it seems to be larger than the screen and requires horizontal browsing:
image

**Comment by Ozerich** [comment]
Fixed

**Pull request merged**